static/js/src/entry/donate/Thermometer.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="thermometer">
```

Comment:
This will break if the AJAX call fails. So I'd have a `v-if` check in here [like Wall component does](https://github.com/texastribune/salesforce-stripe/blob/master/static/js/src/entry/circle/Wall.vue#L3) to prevent that.